contracts/external/policies/UniActionPolicy.sol

```diff
@@ -76,7 +83,7 @@ contract UniActionPolicy is IActionPolicy {
     {
         require(status[id][msg.sender][account] == Status.Live);
         ActionConfig storage config = actionConfigs[id][msg.sender][account];
-        require(value <= config.valueLimit);
+        require(value <= config.valueLimitPerUse);
         ParamRule[16] memory rules = config.paramRules.rules;
```

Comment:
could maybe use a storage  reference here?

Reply:
great catch, thank you! fixed